assets/onboarding/index.jsx

```diff
@@ -1,24 +1,41 @@
-import { Stepper } from '@woocommerce/components';
 import { render } from '@wordpress/element';
-import { useFullScreen } from './use-fullscreen';
+import { useWpAdminFullscreen } from '../react-hooks';
 import { steps } from './steps.jsx';
+import QueryStringRouter, { Route } from './query-string-router';
+import Navigation from './navigation';
 
-function SenseiOnboardingPage() {
-	useFullScreen( [ 'sensei-color', 'sensei-onboarding__page' ] );
+/**
+ * Param name used to route the onboarding wizard.
+ */
+const PARAM_NAME = 'step';
 
-	const currentStep = steps[ 0 ];
+/**
+ * Sensei onboarding page.
+ *
+ * @return {Object} Sensei onboarding page.
```

Comment:
I wonder if we can ditch the @return tags for React components. It will usually be just a duplicate of the short description. Eslint doesn't seem to mind if it's there.

Reply:
That's true! It makes sense to me. https://github.com/Automattic/sensei/pull/3085/commits/48146d8744cddfabf06c32d3809b8c5aff149c0d

Reply:
I kept it only in the Route component where an explanation would be worthwhile.